### PR TITLE
Include human-written enumeration on holdings

### PR DESCRIPTION
**Pull request by alexwlchan**
For https://github.com/wellcomecollection/platform/issues/5477

### CODE REVIEW

File: pipeline/transformer/transformer_sierra/src/test/scala/weco/pipeline/transformer/sierra/transformers/SierraHoldingsEnumerationTest.scala [hunk 1]
```diff
@@ -918,6 +918,17 @@ class SierraHoldingsEnumerationTest
     }
   }
 
+  it("finds a human-written description in field tag h") {
```
Comment (paul-butcher) [inline pipeline/transformer/transformer_sierra/src/test/scala/weco/pipeline/transformer/sierra/transformers/SierraHoldingsEnumerationTest.scala:921]: For completeness, I would add a test containing both human and automatic enumerations.

I think the suite would currently pass if the code were changed to first try fetching one, then fetching the other if the first comes out empty.

Comment (alexwlchan) [reply pipeline/transformer/transformer_sierra/src/test/scala/weco/pipeline/transformer/sierra/transformers/SierraHoldingsEnumerationTest.scala:921]: Good suggestion, will add.

File: pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/SierraHoldingsEnumeration.scala [hunk 2]
```diff
@@ -26,11 +26,28 @@ import scala.util.{Failure, Success, Try}
...
+  // Field tag h is used by librarians to describe the contents of our holdings
+  // when they wrote the description themselves.  Note that 853/863 will also
+  // have field tag h, so we want to filter to cases where it's just an h and
+  // a textual description.
+  //
+  // See https://documentation.iii.com/sierrahelp/Content/sril/sril_records_varfld_types_holdings.html
+  //
+  private def getHumanWrittenEnumeration(varFields: List[VarField]): List[String] =
+    varFields
+      .filterNot(vf => vf.marcTag.isDefined)
```
[the remark this answers is not in the document]
Comment (alexwlchan) [reply pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/SierraHoldingsEnumeration.scala:45]: …how did I miss that, let me try again

**Review by paul-butcher — APPROVED** [review]

**Comment by alexwlchan** [comment]
This is deployed, and I've kicked off a reindex for the IDs it affects.
